Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -65,11 +65,15 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.tableView.accessibilityIdentifier = "Bookmarks List"
+        self.setupBookmarkSource()
```

Comment:
1. I agree. I just wasn't sure if there was any behavior that _required_ for it to be called in `viewwillappear`. For example some kind of reuse somewhere. So to be safe I just added it to viewDidLoad.
2. yup it is most likely related to that. Another way to get around this is to simply manually call viewwillAppear when we create the Panel. This approach is probably the simplest 
3. Dunno. This is the reason why I didn't move everything out of viewWillAppear.